client/core/wallet.go

```diff
@@ -16,6 +16,7 @@ import (
 type xcWallet struct {
 	asset.Wallet
 	connector *dex.ConnectionMaster
+	Account   string
```

Comment:
Only purpose is logging and debugging, but it never changes and adds little additional complexity.